src/app/pages/component-category-list/component-category-list.html

```diff
@@ -1,5 +1,6 @@
 <div class="docs-component-category-list">
   <md-card
+      role="link"
```

Comment:
I think a better approach for this one would be to wrap the entire card inside of an anchor element.